src/panels/lovelace/cards/hui-logbook-card.ts

```diff
@@ -229,23 +237,27 @@ export class HuiLogbookCard extends LitElement implements LovelaceCard {
     const lastDate = this._lastLogbookDate || hoursToShowDate;
     const now = new Date();
 
-    const newEntries = await getLogbookData(
-      this.hass,
-      lastDate.toISOString(),
-      now.toISOString(),
-      this._configEntities!.map((entity) => entity.entity).toString(),
-      true
-    );
-
-    const logbookEntries = this._logbookEntries
-      ? [...newEntries, ...this._logbookEntries]
-      : newEntries;
-
-    this._logbookEntries = logbookEntries.filter(
-      (logEntry) => new Date(logEntry.when) > hoursToShowDate
-    );
-
-    this._lastLogbookDate = now;
+    try {
```

Comment:
Only put the try catch around the `getLogbookData` call

Reply:
I have narrowed now the try/catch scopes or did you mean to split up the `Promise.all()` calls into separate ones so we can really only try/catch `getLogbookDate()`? Then we would loose the fail-fast behavior.